angular/src/directives/navigation/ion-router-outlet.ts

```diff
@@ -233,6 +233,12 @@ export class IonRouterOutlet implements OnDestroy, OnInit {
       this.updateActivatedRouteProxy(cmpRef.instance, activatedRoute);
     } else {
       const snapshot = (activatedRoute as any)._futureSnapshot;
+
+      if (snapshot.component) {
```

Comment:
Can we add a comment that explains why we check for `snapshot.component`? Is it that only standalone components always have this populated?